src/Compression/CompressionFactory.cpp

```diff
@@ -179,6 +179,7 @@ void registerCodecDoubleDelta(CompressionCodecFactory & factory);
 void registerCodecGorilla(CompressionCodecFactory & factory);
 void registerCodecEncrypted(CompressionCodecFactory & factory);
 void registerCodecFPC(CompressionCodecFactory & factory);
+void registerCodecGCD(CompressionCodecFactory & factory);
```

Comment:
Kind of unrelated, but I'd appreciate if you could wrap l. 171 in `#ifdef ENABLE_QPL_COMPRESSION`.